.github/workflows/cd.yml

```diff
@@ -1,50 +1,101 @@
 name: Create docker images
 
-on: [create]
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think the list of branches should be dropped, so a docker image will be created for people's forks (but on their own account)